Manifests/ManagedPreferencesApplications/com.google.drivefs.settings.plist

```diff
@@ -320,7 +362,7 @@ The payload organization for a payload need not match the payload organization i
 			<key>pfm_name</key>
 			<string>ForceBrowserAuth</string>
 			<key>pfm_note</key>
-			<string>If your organization uses security keys or SSO, this setting may resolve sign-in problems. NOTE: Undocumented key.</string>
+			<string>NOTE: Undocumented key. If your organization uses security keys or SSO, this setting may resolve sign-in problems.</string>
```

Comment:
If you're changing the order here, remember that both PC and iPE already display a note title before `pfm_note` texts, so having the text start with "NOTE: " will double that.